src/parser/filters/normalizeDescriptor.js

```diff
@@ -75,6 +79,10 @@ function isBass(intervals) {
 	return hasExactly(intervals, ['1']);
 }
 
+function isAlteredChord(intents) {
+	return intents.alt;
```

Comment:
where do you set intents.alt?

Reply:
I always get 
	"intents": {
		"major": true,
		"eleventh": false
	},
and can't manage to get an alt intent

Reply:
Maybe that is because of the outdated bundle?

https://github.com/no-chris/chord-symbol/blob/e2f923938770bb7845fda36b59db4c28792780b0/src/parser/filters/parseDescriptor.js#L303

Reply:
oh, yes, ok it works now.